src/app/home/shared/forms/transaction-form/transaction-form.component.scss

```diff
@@ -81,3 +81,51 @@ $top-container: 2rem;
   flex-direction: column;
   justify-content: flex-end;
 }
+
+input[type=checkbox] + label {
+  display: block;
+  margin: 3rem 0 0 1rem;
+  cursor: pointer;
+  padding: 0.2rem;
+}
+
+input[type=checkbox] {
+  display: none;
+}
+
+input[type=checkbox] + label:before {
+  content: "\2714";
+  border: 0.1em solid $light-gray;
+  border-radius: 0.2rem;
+  display: inline-block;
+  width: 1rem;
+  height: 1rem;
+  padding-left: 0.3rem;
+  padding-right: 0.1rem;
+  padding-bottom: 0.4rem;
+  margin-right: 0.4rem;
+  vertical-align: center;
+  color: transparent;
+  transition: .2s;
+}
```

Comment:
I guess it might have been more elegant, if you would just create a component or directive for the checkbox.